content/shared/js/skipto.js

```diff
@@ -2648,14 +5305,82 @@ $skipToId [role="menuitem"].hover .label {
         smallButtonLabel: 'SkipTo',
         altLabel: 'Alt',
         optionLabel: 'Option',
+        shortcutLabel: 'shortcut',
         buttonShortcut: ' ($modifier+$key)',
-        altButtonAriaLabel: 'Skip To Content, shortcut Alt plus $key',
-        optionButtonAriaLabel: 'Skip To Content, shortcut Option plus $key',
+        buttonAriaLabel: '$buttonLabel, $shortcutLabel $modifierLabel + $key',
+
+        // Page navigation flag and keys
+        shortcutsSupported: 'true', // options: true or false
+        shortcuts: 'enabled',  // options: disabled and enabled
+        shortcutHeadingNext: 'h',
+        shortcutHeadingPrevious: 'H',
+        shortcutHeadingH1: '1',
+        shortcutHeadingH2: '2',
+        shortcutHeadingH3: '3',
+        shortcutHeadingH4: '4',
+        shortcutHeadingH5: '5',
+        shortcutHeadingH6: '6',
+
+        shortcutRegionNext: 'r',
+        shortcutRegionPrevious: 'R',
+        shortcutRegionMain: 'm',
+        shortcutRegionNavigation: 'n',
+        shortcutRegionComplementary: 'c',
+
+        shortcutsGroupEnabledLabel:  'Shortcuts: Enabled',
+        shortcutsGroupDisabledLabel: 'Shortcuts: Disabled',

```

Comment:
I was a little disoriented by the invisible `<div role="separator">Shortcuts: Disabled</div>` that’s exposed in the a11y tree between “Help improve this page” and “About SkipTo.js”? Is this meant to be discoverable to assistive tech users?